CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ An error is now logged instead, and extraction continues after a bad line is enc
 - Keepalive events will now continue to fire after cluster restarts.
 - Fixed a panic in the dashboardd shutdown routine.
 - Fixed a bug where deleting a non-existent entity with sensuctl would not return an error.
+- Check TTL failure events are now much more reliable, and will persist even
+in the presence cluster member failures and cluster restarts.
```

Comment:
You'll probably need to merge/rebase master and ensure the entry is in the unreleased section!